kernel/ap_start/src/lib.rs

```diff
@@ -32,26 +24,26 @@ pub static AP_READY_FLAG: AtomicBool = AtomicBool::new(false);
 
 /// Temporary storage for transferring allocated `Stack`s from 
 /// the main bootstrap processor (BSP) to the AP processor being booted in `kstart_ap()` below.
-static AP_STACKS: MutexIrqSafe<BTreeMap<u8, NoDrop<Stack>>> = MutexIrqSafe::new(BTreeMap::new());
+static AP_STACKS: MutexIrqSafe<BTreeMap<u32, NoDrop<Stack>>> = MutexIrqSafe::new(BTreeMap::new());
 
```

Comment:
unfortunately not, because the origin of this value is an ACPI system table, which is a raw value that must be used in order to actually bring up this particular APIC/CPU. So we couldn't possibly create a `CpuId` or `ApicId` at this point because that APIC/CPU wouldn't yet exist.